File: companies/random/backend.php
<?php

  // Your goal is to count how many items exist that have an age equal to or greater than 50, and print this final value.

  $ch = curl_init('https://coderbyte.com/api/challenges/json/age-counting');
  curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
  curl_setopt($ch, CURLOPT_HEADER, 0);
  $data = curl_exec($ch);
  curl_close($ch);

  $json_data = json_decode($data, true);
  $items = explode(', ', $json_data['data']);
  $count = array_reduce($items, function ($count, $item) {
    if (strpos($item, 'age=') !== false) {
      $age = explode('=', $item)[1];
      if ($age >= 50) return $count + 1;
    }
    return $count;
  }, 0);

  print_r($count); // 128

?>

File: companies/random/backend.py
import pandas as pd
import requests

res = requests.get(
    'https://coderbyte.com/api/challenges/json/age-counting'
).json()


data = res['data']
pt = data.split(", ")
cnt = 0
for i in range(0,len(pt),2):
    key,age = pt[i].split("=")[-1],pt[i+1].split("=")[-1]
    if(int(age)>=50): cnt +=1;
print(cnt)